### ISSUE TITLE
Ostree Version: 40.20240624.0 (2024-06-24T17:14:39Z) seems to be defective

### ISSUE THREAD

**Issue by Moshugan** [ublue-os/bazzite#1281]
### Describe the bug

Computer runs slow, lags, apps load slowly, streaming video lags and sound pops etc. under the ostree:0 which is Version: 40.20240624.0 (2024-06-24T17:14:39Z) currently. The issue started after I last shutdown the system. I'm not sure at which point the update broke something. When I choose ostree:1 which is Version: 40.20240619.0 (2024-06-20T01:06:19Z) everything works as it should, smooth and quick.

### What did you expect to happen?

I expected the system to run the same on Version: 40.20240624.0 as it did last time I used it, and as it does run on Version: 40.20240619.0 if I choose it now.

### Output of `rpm-ostree status`

```shell
State: idle
Deployments:
  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-nvidia:stable
                   Digest: sha256:f99281982c87a4d18bfdba2f2c06698a5017028af2e893c617546643ba542071
                  Version: 40.20240624.0 (2024-06-24T17:14:39Z)
                     Diff: 233 upgraded, 51 removed, 37 added

● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-nvidia:stable
                   Digest: sha256:376ffb04e940fce1390027c1ffa65662e8521d209dc7f5fbd1bc4ba3c235f718
                  Version: 40.20240619.0 (2024-06-20T01:06:19Z)
```

### Hardware

Output from fastfetch:
- bazzite-nvidia:stable
- Fedora Linux 40 (Kinoite) x86_64
- Linux 6.9.4-201.fsync.fc40.x86_64
- 34 mins
- Spawned on Jun 10 2024
- MS-7C02 (1.0)
- AMD Ryzen 5 3600 (12) @ 3.60 GHz
- NVIDIA GeForce RTX 3070 Lite Hash]
- 5.98 GiB / 15.56 GiB (38%)
- 248.30 GiB / 930.50 GiB (27%) - b]
- 560x1440 @ 144Hz *
- 1024x768 @ 85Hz
- Microsoft X-Box 360 pad
- Microsoft X-Box 360 pad 0
- KDE Plasma 6.0.5
- KWin (X11)
- bash 5.2.26
- ptyxis-agent
- 2714 (rpm), 59 (flatpak-system), )

**Comment by Moshugan** [comment]
Sorry if this wasn't the proper way and place to report such an issue, but I couldn't find anything about it on any platform I tried searching on.

**Comment by SContent44** [comment]
## tldr
I had the same issues in the same timeframe and it seems to have been solved for me with BIOS update and/or reset.

## Description of issue
> Computer runs slow, lags, apps load slowly, streaming video lags and sound pops etc

Just wanted to chime in to say I was experiencing what sounds to be the same issues for the last few days. I had this issue with both: 
- 40-stable-20240627,
- 40-stable-20240624

And I had to rollback to **40.20240621.0** to not have the performance issues.



The main thing I that along with the performance issues is that the CPU usage looked really strange with all the cores seeming like they were idling around 40% usage.

Looking in `journalctl` the main thing that stood out to me when using the "bad" images was the following:

```
Jun 28 17:41:44 fedora kernel: processor cpu0: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu1: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu2: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu3: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu4: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu5: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu6: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu7: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu8: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu9: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu10: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu11: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu12: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu13: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu14: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: processor cpu15: min_freq(0) or max_freq(0) or nominal_freq(0) value is incorrect, check _CPC in ACPI tables
Jun 28 17:41:44 fedora kernel: amd_pstate: failed to register with return -19
```

Since ACPI was mentioned (and I have no idea what I'm really doing) I tried to take some variables out of the equation so I updated my BIOS to the latest version. Asrock x370 Gaming X 6.00 through to to 10.08 [BETA]. 

After that I tried the latest image again and it is now working. I don't know if it was the update or the BIOS resets that accompanied it.

## After BIOS update / resets

Now its performing like normal and I am no longer seeing the strange CPU usage or the above logs. 


## Output of rpm-ostree status
Version 40.20240621.0 which is pinned is what I had to rollback to to not have the performance issues.
```
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-gnome-nvidia:stable
                   Digest: sha256:26e7df3e555a1ad6fb7824c01ae43130bc9447d63274810c2f3d7455374f198d
                  Version: 40.20240625.0 (2024-06-27T02:01:58Z)
          LayeredPackages: sunshine

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-gnome-nvidia:stable
                   Digest: sha256:38eb23a1139dcd658625672dc9728a96f47dad8096f2820da5463c9b30099a72
                  Version: 40.20240621.0 (2024-06-21T05:11:25Z)
          LayeredPackages: sunshine
                   Pinned: yes
```


## Hardware

Output from fastfetch (post it being fixed with stuff trimmed):
- bazzite-gnome-nvidia:stable
- Fedora Linux 40 (Silverblue) x86_64
- Linux 6.9.6-201.fsync.fc40.x86_64
- X370 Gaming X
- AMD Ryzen 7 3700X (16) @ 4.43 GHz
- NVIDIA GeForce RTX 2060 SUPER [Discrete]
- 5.41 GiB / 31.26 GiB (17%)
 - 752.12 GiB / 929.93 GiB (81%) - btrfs [Read-only]
- GNOME 46.2
- Mutter (X11)
- bash 5.2.26
- ptyxis-agent
- 2413 (rpm), 104 (flatpak-system), 55 (flatpak-user)


_

I hope any of that will help you since it was very frustrating not having any clear idea and this issue post was very helpful in seeing that this wasn't potentially happening to just me.

**Comment by gbagnoli** [comment]
This is happening to me as well - everything past 40.20240621.0 is laggish, slow, unusable, including the 3.5.0 last released.

## Hardware Information:
- **Hardware Model:**                              Micro-Star International Co., Ltd. MS-7B79
- **Memory:**                                      16.0 GiB
- **Processor:**                                   AMD Ryzen™ 5 3600 × 12
- **Graphics:**                                    NVIDIA GeForce RTX™ 2060 SUPER
- **Disk Capacity:**                               1.0 TB

## Software Information:

- **OS Name:**                                     Fedora Linux 40.20240621.0 (Bazzite GNOME)
- **OS Type:**                                     64-bit
- **GNOME Version:**                               46
- **Windowing System:**                            X11
- **Kernel Version:**                              Linux 6.9.4-201.fsync.fc40.x86_64

**Comment by gbagnoli** [comment]
ok, I also solved with a bios update of the motherboard (I had a 2019 bios, updated to the last one). Resetting the old bios did not work.

```
OS: Fedora Linux 40 (Silverblue) x86_64
Host: MS-7B79 (3.0)
Kernel: Linux 6.9.6-201.fsync.fc40.x86_64
Uptime: 4 hours, 19 mins
Packages: 2371 (rpm), 78 (flatpak-system), 18 (flatpak-user)
Shell: bash 5.2.26
Display (DELL P3222QE): 3840x2160 @ 60Hz
DE: GNOME 46.2
WM: Mutter (Wayland)
WM Theme: Bazzite
Theme: adw-gtk3-dark [GTK2/3/4]
Icons: Adwaita [GTK2/3/4]
Font: Noto Sans (11pt) [GTK2/3/4]
Cursor: Adwaita (24px)
Terminal: ptyxis-agent
CPU: AMD Ryzen 5 3600 (12) @ 4.21 GHz
GPU: NVIDIA GeForce RTX 2060 SUPER [Discrete]
Memory: 4.83 GiB / 15.53 GiB (31%)
Swap: 2.55 GiB / 4.00 GiB (64%)
Disk (/sysroot): 148.83 GiB / 929.93 GiB (16%) - btrfs [Read-only]
```

**Comment by closeke** [comment]
Thanks for this. Confirming that a bios update did the trick for me as well. I didn't realize how old my BIOS was. It was on version F11 from 2019 and is now on version F39 from a few months ago.

**Comment by Moshugan** [comment]
Thanks for all the replies! I'm returning to this post a bit late, but I appreciate the work you've all done. Updating the BIOS helped me too! Although I'm having an issue with sound in current versions of Bazzite. I'm probably going to open a ticket for it.
